tests/dvslib/dvs_mirror.py

```diff
@@ -0,0 +1,100 @@
+class DVSMirror(object):
+    def __init__(self, adb, cdb, sdb, cntrdb, appdb):
+        self.asic_db = adb
+        self.config_db = cdb
+        self.state_db = sdb
+        self.counters_db = cntrdb
+        self.app_db = appdb
+
+    def create_span_session(self, name, dst_port, src_ports=None, direction="BOTH", queue=None, policer=None):
+        mirror_entry = {"type": "SPAN"}
+        if dst_port:
+            mirror_entry["dst_port"] = dst_port
+
+        if src_ports:
+            mirror_entry["src_port"] = src_ports
+        # set direction without source port to uncover any swss issues.
```

Comment:
Not sure I understand this, what are we trying to uncover?

Reply:
In mirrororch we check both src/direction is set before configure port mirror session.
If direction is only set then it shouldnt do any port mirror config and also session has to be active.(based on other conditions)
This setting just makes sure that setting direction only doesn't break anything :)